src/main/resources/langs/fr_FR.yml

```diff
@@ -82,13 +82,13 @@ administration:
     aboutToResetEverything: "You're about to reset the XP for everyone on the entire server. If you're sure you wanna continue, run the command to completely reset the servers XP.\n\n```:command server-reset :token```\nThis command will only work for 60 seconds, after that the token becomes invalid and the command has to be ran again to get a new token."
     aboutToSyncEverything: "You're about to sync the XP with the global leaderboard, this will effectively revert any custom XP changes made on the entire server. If you're sure you wanna continue, run the command to completely sync the servers XP.\n\n```:command server-sync :token```\nThis command will only work for 60 seconds, after that the token becomes invalid and the command has to be ran again to get a new token."
     failedToSaveChanges: "Failed to save the changes to the database, please try again, if the error continues to happen, please contact support on the official [AvaIre support server](https://avairebot.com/support)."
-    mustBeAnAdmin: "You must be a server admin to use this action."
+    mustBeAnAdmin: "Vous devez être un administrateur du serveur pour utiliser cette action."
     invalidSecurityToken: "Invalid security token given, please provide the correct security token to reset all the servers XP."
-    warning: "Warning!"
+    warning: "Attention !"
```

Comment:
Possibly autotranslated (Just remove the " " between the ``Attention`` and ``!``